app/lib/document_types/form13614C.rb

```diff
@@ -8,7 +8,7 @@ def relevant_to?(intake)
       end
 
       def key
-        "F13614C 2020"
+        "F13614C"
```

Comment:
we probably need a migration path to ensure existing docs in the database have this `key` eventually

I dunno what's a sane way... If we want to avoid errors we would probably have to support both the new and the old-style `key` for this doctype class while migrating the data

Reply:
@tgrathwell what do you mean by support both new keys? I'm trying to find places in the code where it would be disrupted if we changed the key but I can not but maybe I'm looking for the right thing?